compose/ui/ui-test-manifest/integration-tests/testapp/build.gradle

```diff
@@ -22,12 +22,12 @@ plugins {
 }
 
 dependencies {
-    kotlinPlugin(project(":compose:compiler:compiler"))
+    kotlinPlugin(projectOrArtifact(":compose:compiler:compiler"))
```

Comment:
I think it is best for us to included compose:compiler:compiler, otherwise we'll likely have a lot of breakages as it changes.

Reply:
Done